aiida/tools/importexport/config.py

```diff
@@ -21,6 +21,9 @@
 IMPORTGROUP_TYPE = GroupTypeString.IMPORTGROUP_TYPE.value
 DUPL_SUFFIX = ' (Imported #{})'
 
+# The name of the subfolder in which the node files are stored
+NODES_EXPORT_SUBFOLDER = 'nodes'
```

Comment:
You might also want to use this new constant in `aiida/tools/importexport/migration/v03_to_v04.py` and `aiida/common/archive.py` where it is also used

Reply:
Found it.

It seems to me that the functions `export_zip`, `export_tar`, and `export_tree` are also only used in import/export - move it? :sweat_smile:

Reply:
Might as well move the whole file then. Only `Archive` class would be remaining which is also only used in export/import. While you're at it might also replace `CorruptArchive` then with `ArchiveIntegrityError` or vice versa.